### PR TITLE
Using env vars for getting the metadata details

### PR DESCRIPTION
**Pull request by yussufsh**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

This will help with running the driver tests locally. Currently, it relies on k8s cluster node.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/driver/controller.go [hunk 1]
```diff
@@ -91,11 +91,15 @@ var (
...
 // it will print stack trace and osexit if failed to create the service
 func newControllerService(driverOptions *Options) controllerService {
-	var (
-		cloudInstanceId string
-		zone            string
-	)
-	if driverOptions.cloudconfig != "" {
+	var cloudInstanceId, zone string
+
+	cID := os.Getenv("POWERVS_CLOUD_INSTANCE_ID")
```
Comment (mkumatag) [inline pkg/driver/controller.go:96]: Please add a comment here with all the detail why we are adding these variables

File: pkg/driver/node.go [hunk 2]
```diff
@@ -75,13 +75,25 @@ type nodeService struct {
 // newNodeService creates a new node service
 // it will print stack trace and osexit if failed to create the service
 func newNodeService(driverOptions *Options) nodeService {
-	klog.V(4).Infof("retrieving node info from metadata service")
-	metadata, err := cloud.NewMetadataService(cloud.DefaultKubernetesAPIClient, driverOptions.kubeconfig)
-	if err != nil {
-		klog.Fatalf("Failed to get metadata service: %v", err)
+	var cloudInstanceId, zone, instanceID string
+
+	cID := os.Getenv("POWERVS_CLOUD_INSTANCE_ID")
```
Comment (mkumatag) [inline pkg/driver/node.go:80]: Please add a comment here with all the detail why we are adding these variables

**Review by mkumatag — CHANGES_REQUESTED** [review]
And also add some document around what we are trying to achieve for these modifications like some text talking about integration tests and how to run them etc..

**Comment by yussufsh** [comment]
> And also add some document around what we are trying to achieve for these modifications like some text talking about integration tests and how to run them etc..

I will add the docs for running IT when the framework is ready.
For now, added comments. PTAL again.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**